packages/cli/src/commands/dns/rm.ts

```diff
@@ -103,7 +103,7 @@ function getDeleteTableRow(domainName: string, record: DNSRecord) {
   return [
     record.id,
     chalk.bold(
-      `${recordName} ${record.type} ${record.value} ${record.mxPriority || ''}`
+      `${recordName} ${record.recordType} ${record.value} ${record.mxPriority || ''}`
```

Comment:
Thanks for the PR!

However, this will fail to build because the interface doesn't have `recordType` property:

https://github.com/vercel/vercel/blob/main/packages/cli/src/types.ts#L152-L166

Furthermore, there is no `recordType` property documented for Domain or DNS

https://vercel.com/docs/rest-api#endpoints/dns

https://vercel.com/docs/rest-api#endpoints/domains

I'll defer to @mglagola

Reply:
Righto, makes sense! That's why I thought maybe something is up with the `v5` endpoint? It's definitely returning `recordType` instead of `record`:

```
curl "https://api.vercel.com/v5/domains/records/rec_45aa1cd08e614b7fce9acecd" \
     -H 'Authorization: Bearer [testingToken]'
```
Returns:
```
{"createdAt":1631766102686,"creator":"z9nQ0UbzQf52vfYODE78vvVk","domain":"jacobford.com","id":"rec_45aa1cd08e614b7fce9acecd","name":"waterboa","recordType":"CNAME","value":"starman.fathomdns.com.","ttl":60}%
```

Reply:
@unitof Thanks for finding this. This was an api bug - the endpoint responds with `type` now.

Reply:
Confirmed! Thanks @mglagola.